src/components/SocialBtns/facebookSDK.js

```diff
@@ -0,0 +1,32 @@
+ /* eslint-disable */
```

Comment:
Make sure to enable eslint at the end of the file

http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments